server/controllers/submission.js

```diff
@@ -29,3 +33,41 @@ exports.createSubmission = asyncHandler(async (req, res) => {
     res.status(500).json(err);
   }
 });
+
+exports.getSubmissionsByUser = asyncHandler(async (req, res) => {
```

Comment:
You can combine the logic for both controllers into one, that way it's easier to send a request from the FE as we would have a single endpoint